File: C/c005.py
input_lines = int(raw_input())
for i in xrange(input_lines):
  s = raw_input().rstrip().split('.')
  if len(s) != 4:
    print "False"
    continue
  ans = "True"
  for ip in s:
    if ip.isdigit()==False or int(ip)<0 or int(ip)>255:
      ans = "False"
      break
  print ans

File: C/ruby/c005.rb
input_lines = gets.to_i
input_lines.times{
  s = gets.chomp.split(".")
  if s.length != 4 then
    print "False", "\n"
    next
  end
  ans = "True"
  for ip in s
    if ip !~ /\d+/ or ip.to_i < 0 or ip.to_i > 255 then
      ans = "False"
      break
    end
  end
  print ans, "\n"
}
